src/Schema/SchemaInterface.php

```diff
@@ -206,6 +206,10 @@ interface SchemaInterface extends ConstraintSchemaInterface
      * Define the abstract column type as `binary`.
      */
     public const TYPE_BINARY = 'binary';
+    /**
```

Comment:
As it is not a global `type` that behaves the same for all `dbms`, i think it would be a good idea to map it in `db-pgsql`, and each `dbms` will have its own behavior, as it should be.